docs/installation/integrations/laravel.md

```diff
@@ -0,0 +1,90 @@
+---
+category: integrations
+order: 70
+menu-title: Laravel
+---
+
+# Compatibility with Laravel
+
+As a pure JavaScript/TypeScript application, CKEditor 5 will work inside any environment that supports such components. While we do not offer official integrations for any non-JavaScript frameworks, you can include a custom build of CKE5 in a non-JS framework of your choice, for example, the PHP-based [Laravel](https://laravel.com/).
+
+The core of the integration is preparing a build of CKEditor 5 with the [online builder](https://ckeditor.com/ckeditor-5/online-builder/), and then importing it into your Laravel project.
+
+## Preparing a build
+
+The easiest way of preparing a custom build of CKEditor 5 is to use our [online builder](https://ckeditor.com/ckeditor-5/online-builder/). It is a powerful tool that lets you efortlessly create a rich text editor that is custom-tailored to your needs. With the online builder, you can choose the desired editor type, plugins, configure the toolbar, and choose the UI language for your editor.
+
+You can learn more about creating custom builds of CKE5 via the online builder in our {@link installation/getting-started/quick-start-other#creating-custom-builds-with-online-builder Customized installation guide}.
+
+## Setting up the project
+
+For the purpose of this guide, we will use a basic Laravel project created with [Composer](https://getcomposer.org/). You can refer to the [Laravel documentation](https://laravel.com/docs/10.x/installation) to learn how to set up a project in the framework.
+
+Once the project has been prepared, create an `assets/vendor` directory in the existing `public` directory in your app. Your folder structure should resemble this one:
+
+````
+├── app
+├── bootstrap
+├── config
+├── database
+├── public
+│   ├── assets
+|      └── vendor
+│   ├── .htaccess
+│   ├── favicon.ico
+│   ├── index.php
+│   ├── robots.txt
+│   └── webpack.config.js
+├── resources
+├── routes
+└── ...
+````
+
+## Integrating the build in your Laravel project
+
+Once you have your custom build of the editor ready, and the Laravel project has been set up, extract the `.zip` folder obtained from the online builder and place it in the `assets/vendor` directory created in the previous step. Your folder structure should now look like this:
+
+````
+├── app
+├── bootstrap
+├── config
+├── database
+├── public
+│   ├── assets
+|      ├── vendor
+|          └── ckeditor5
```

Comment:
While following the guide I did not encounter any bugs, but one small issue may arise when the user downloads the .zip from online builder. The end user needs to change the folder name or change the script’s path. For instance, the original folder name "/ckeditor5-38.0.1-8s0tbeh90pmq" should be changed to "/ckeditor5" for the project to function correctly.

Reply:
Good point! I included an info-box that clarifies this step.